xcape/cape_fortran.py

```diff
@@ -1,14 +1,26 @@
 
-def cape(p_2d, t_2d, td_2d, p_s, t_s, td_s, source, ml_depth, adiabat, pinc, type_grid):
-    from xcape import CAPE_CODE
+def cape(p_2d, t_2d, td_2d, p_s, t_s, td_s, pres_lev_pos, source, ml_depth, adiabat, pinc, type_grid):
+    
+                
```

Comment:
pres_lev_pos is an array like with pressure level position 
it has to be in Fortran indices (1:...) for now. We could leave it to be in python and then just add 1 before passing it to the fortran code.